components/SubTrackDetails/SubTrackDetails.scss

```diff
@@ -0,0 +1,52 @@
+@import "topcoder/tc-includes";
+
+
+.SubTrackDetails{
+    
+    .heading {
+         height:20px;
+        
+        .title {
+            font-size: 15px;
+            line-height: 18px;
+            margin-left: 11px;
+            color: #FFFFFF;
```

Comment:
We can use the `$white` variable for the color, which comes from the `@import` at the top